interpreter/planner.go

```diff
@@ -255,6 +254,15 @@ func (p *planner) planCall(expr *exprpb.Expr) (Interpretable, error) {
 		len(oRef.GetOverloadId()) == 1 {
 		oName = oRef.GetOverloadId()[0]
 	}
+	// Try to find the specific function by overload id.
+	var fnDef *functions.Overload
+	if oName != "" {
+		fnDef, _ = p.disp.FindOverload(oName)
+	}
+	// If the overload id couldn't resolve the function, try the simple function name.
+	if fnDef == nil {
+		fnDef, _ = p.disp.FindOverload(fnName)
+	}
```

Comment:
Nit: You can delay the computation until after the specialized functions are accounted for. Won't make a big performance difference unless you make a heavy use of the built-in ones.